### PR TITLE
Update pubspec.yaml

### PR DESCRIPTION
**Pull request by saprahits**
## Connection with issue(s)

Close #???

Connected to #???

## To Do

- [ ] Read [contributing guide](https://github.com/flutter-form-builder-ecosystem/.github/blob/main/CONTRIBUTING.md)
- [ ] Check the original issue to confirm it is fully satisfied
- [ ] Add solution description to help guide reviewers
- [ ] Add unit test to verify new or fixed behaviour
- [ ] If apply, add documentation to code properties and package readme

### CODE REVIEW

**Comment by flikkr** [comment]
@saprahits Can you update flutter_form_builder dependency to > 10.0.0

**Comment by deandreamatias** [comment]
Blocked until merge #109

**Comment by saprahits** [comment]
> @saprahits Can you update flutter_form_builder dependency to > 10.0.0

@flikkr this is the latest version: https://pub.dev/packages/flutter_form_builder

File: pubspec.yaml [hunk 1]
```diff
@@ -15,11 +15,12 @@ dependencies:
...
   flutter_form_builder: ^9.3.0
-  flutter_colorpicker: ^1.0.3
+  flutter_colorpicker: ^1.1.0
   dropdown_search: ^5.0.6
-  signature: ^5.4.0
+  signature: ^5.5.0
   intl: ^0.19.0
-  flutter_typeahead: ^4.7.0
+  flutter_typeahead: ^5.2.0
+  flutter_form_builder: ^9.3.0
```
Comment (deandreamatias) [inline pubspec.yaml:23]:
```suggestion

```

**Pull request closed without merging**